src/global.css

```diff
@@ -12,3 +15,17 @@
   font-family: "Poppins";
   src: url("./assets/fonts/Poppins/Poppins-Regular.ttf");
 }
+
+@layer mas-custom-styles {
+  .default-input {
```

Comment:
Should we think about a naming conventions in case other components need a similar custom class ?

Reply:
Sure! 

For now I would say we can use **default** (or similar) in the beginning of the class. In a worst case if we decide to change it after we can replace across the app easily.

Reply:
sounds good !